build-system/tasks/visual-diff/index.js

```diff
@@ -241,6 +240,12 @@ async function launchBrowser() {
   browser_ = await puppeteer.launch(browserOptions)
       .catch(err => log('fatal', err));
 
+  // Every action on the browser or its pages adds a listener to the
+  // Puppeteer.Connection.Events.Disconnected event. This is a temporary
+  // workaround for the Node runtime warning that is emitted once 11 listeners
+  // are added to the same object.
```

Comment:
Curious: If this is a temporary workaround, is there a better plan for later?

Reply:
There are multiple bugs in Puppeteer about this, hopefully once this is resolved we can just remove it

Reply:
Can you link the issue for posterity?

Reply:
Weirdly enough the issue has been closed as fixed, but I still get this. I'll dig around, see what comes up